sh_script/integration_tdx.sh

```diff
@@ -0,0 +1,161 @@
+#!/bin/bash
+#
+# Copyright (c) 2022 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+script_path=$(dirname "$0")
+temp_dir=$(mktemp -d)
+nohup_logfile="${temp_dir}/nohup.log"
+
+guest_image="/home/env/guest_img/td-guest-centos8.4.qcow2"
+kernel="/home/env/kernel_img/bzImage"
+cloud_hypervisor_tdx_path="/home/env/cloud-hypervisor/target/release/cloud-hypervisor"
+
+firmware=""
+type="pe"
+
+# Test Configuration Info
+cpus=1
```

Comment:
Should we change this to 2 for smp related testing?

Reply:
Hi Jiang, this is initial version of TDX CI, I will update it this week to contain multiple processors testing

Reply:
I have update 3 configurations: 
cpus 1 + memory 2G
cpus 2 + memory 4G
cpus 3 + memory 8G